btf/traversal.go

```diff
@@ -88,7 +88,7 @@ func (po *postorderIterator) Next() bool {
 }
 
 // walkType calls fn on each child of typ.
-func walkType(typ Type, fn func(*Type)) {
+func walkType(typ Type, fn func(*Type), reverse bool) {
```

Comment:
Not a big fan of the bool, but the only other option I see is just duplicating the function.

Reply:
Fine IMO since the function has a narrow purpose and the bool is mostly self-explanatory. I'd add it to the function doc that it only applies to the order of complex types, though.